templates/deployment-worker.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: airflow
+        tier: worker
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: worker
+        image: eanylin/airflow-helm:1.8.1-1.6.2
```

Comment:
This image, i.e. eanylin/airflow-helm:1.8.1-1.6.2 is actually publicly available (I have uploaded to Docker hub for testing)

I have added the required Dockerfiles to github.com/att-comdev/dockerfiles/airflow as pull request

Have set variable for image in values.yaml